internal/proxy/testdata/TestApply/golden/some_options_set,_configuration_parent_directories_are_created/etc/environment.d/99ubuntu-proxy-manager.conf

```diff
@@ -1,5 +0,0 @@
-### This file was generated by ubuntu-proxy-manager - manual changes will be overwritten
```

Comment:
Do you mind checking this is expected?

Reply:
Ah, indeed, too may changes (as basically every line is changed) for this poor git :)